### PR TITLE
[backend] For Software display the version in addition to the names i…

### LINKED ISSUE #7765: For software observables, in graphs, display the version in addition to the names in parenthesis to avoid wired behaviour
## Description

Display "ubuntu_linux (22.04)" instead of just "ubuntu_linux" to avoid this:

image

### PR DESCRIPTION
**Pull request by ValentinBouzinFiligran**
…n parenthesis (#7765)

### Proposed changes

* software representative update to add version if available

### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/7765

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: opencti-platform/opencti-graphql/src/utils/format.js [hunk 1]
```diff
@@ -122,6 +122,14 @@ export const truncate = (str, limit = DEFAULT_TRUNCATE_LIMIT, withPoints = true)
   return `${trimmedStr.substr(0, Math.min(trimmedStr.length, trimmedStr.lastIndexOf(' ')))}...`;
 };
 
+const formatSoftware = (stixCyberObservable) => {
+  const value = stixCyberObservable.name || stixCyberObservable.cpe || stixCyberObservable.swid || 'Unknown';
```
Comment (Kedae) [inline opencti-platform/opencti-graphql/src/utils/format.js:126]: I think you should get the representative ?

**Review by SarahBocognano — APPROVED** [review]
Tested ✅

File: opencti-platform/opencti-graphql/src/utils/format.js (hunk 1 above)
Comment (ValentinBouzinFiligran) [reply opencti-platform/opencti-graphql/src/utils/format.js:126]: isn't this where representations are generated?

Comment (Kedae) [reply opencti-platform/opencti-graphql/src/utils/format.js:126]: Oh god, I switched from Celine review and I thought you were in the front aswell. Sorry :D

Comment (ValentinBouzinFiligran) [reply opencti-platform/opencti-graphql/src/utils/format.js:126]: is it good for you so ? :D

Comment (Kedae) [reply opencti-platform/opencti-graphql/src/utils/format.js:126]: Yep.

**Pull request merged**